CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of the dpkg_autostart coo
 
 ## Unreleased
 
+- resolved cookstyle error: libraries/provider.rb:13:7 refactor: `Chef/Modernize/ActionMethodInResource`
```

Comment:
From previous reports I need to back this cop out

Reply:
https://github.com/chef/cookstyle/pull/802

Reply:
Shall we go ahead and close this PR then?